app.js

```diff
@@ -1,90 +1,156 @@
 let fs = require('fs'),
```

Comment:
Also I think you should add a test for the following:
`node app.js --files specs/data/test-1.csv`
Because you have fixed that bug without any coverage as well